wave_vision/src/matcher/brute_force_matcher.cpp

```diff
@@ -53,28 +66,140 @@ void BruteForceMatcher::checkConfiguration(
         throw std::invalid_argument(
           "Norm type is not one of the acceptable values!");
     }
+
+    // Check the value of the ratio_test heuristic
+    if (check_config.ratio_threshold < 0.0 ||
+        check_config.ratio_threshold > 1.0) {
+        throw std::invalid_argument(
+          "ratio_threshold is not an appropriate value!");
+    }
+
+    // Check the value of the threshold distance heuristic
+    if (check_config.distance_threshold < 0) {
+        throw std::invalid_argument("distance_threshold is a negative value!");
+    }
+
+    // Only acceptable values are 1, 2, 4, and 8
```

Comment:
So yes, this is already an enum within openCV. The parser really did not like the enum though...